docs/reference/esm.md

```diff
@@ -1,6 +1,12 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/apm/agent/nodejs/current/esm.html
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    apm_agent_node: ga
```

Comment:
There's a note below stating that this is experimental. We don't have an `experimental` lifecycle status. @trentm @david-luna would it be appropriate to label this as `preview` ([list of available lifecycles](https://elastic.github.io/docs-builder/contribute/cumulative-docs/reference/#lifecycle))?

```suggestion
    apm_agent_node: preview
```